worker/app/tasks/base.py

```diff
@@ -52,22 +53,13 @@ def clean_up(self):
 
         # remove working_dir
         working_dir = Path(Settings.working_dir_container).joinpath(self.task_id)
-        shutil.rmtree(working_dir)
-
-    @staticmethod
-    def get_files(working_dir: Path):
-        stats = []
-        description = []
-        for file in working_dir.iterdir():
-            if file.is_dir():
-                continue
-            stats.append({'name': file.name, 'size': file.stat().st_size})
-            description.append('{name} - {size}'.format(name=file.name, size=file.stat().st_size))
-
-        return stats, ', '.join(description)
+        shutil.rmtree(working_dir, ignore_errors=True)
 
     def on_success(self, retval, task_id, args, kwargs):
         self.clean_up()
 
     def on_failure(self, exc, task_id, args, kwargs, einfo):
         self.clean_up()
+
+    def upload_zims(self, remote_working_dir: str, files: list = None, directory: Path = None):
+        return Upload.upload(f'/zim{remote_working_dir}', files, directory)
```

Comment:
Hmmm if this is just one line, I am not sure it is really beneficial to make it a separate function.

Reply:
I'd prefer to keep it even though it's simple as it's gonna be used in every scraper and makes sure we keep that `/zim` prefix in place.